ena_upload/ena_upload.py

```diff
@@ -104,7 +104,7 @@ def check_columns(df, schema, action, dev, auto_action):
             df['scientific_name'] = pd.Series(dtype='str')
     elif schema == 'run':
         optional_columns = ['accession',
-                          'submission_date', 'status', 'file_checksum']
+                          'submission_date', 'status', 'file_checksum', 'unencrypted_checksum']

```

Comment:
```suggestion
                          'submission_date', 'status'
```
I would take out this one, since otherwise it gets printed out as a column into the updated tsv tables even when unencrypted_checksum is not given.

Reply:
oh sure, I overlooked this.